apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Footer/Footer.stories.tsx

```diff
@@ -56,9 +63,18 @@ const Template: Story<
 }
 
 export const Default = Template.bind({})
-Default.args = { hostTitle: null }
+Default.args = { hostTitle: null, chatButtons: [] }
```

Comment:
I think you don't have to add this?
```suggestion
Default.args = { hostTitle: null }
```